docs/gwcs/fits_analog.rst

```diff
@@ -0,0 +1,101 @@
+.. _fits_equivalent_example
+
+FITS Equivalent WCS Example
+===========================
+
+The following example shows how to construct a GWCS object equivalent to
+a FITS imaging WCS without distortion, defined in this FITS imaging header::
+
+  WCSAXES =                    2 / Number of coordinate axes
+  WCSNAME = '47 Tuc     '        / Coordinate system title
+  CRPIX1  =               2048.0 / Pixel coordinate of reference point
+  CRPIX2  =               1024.0 / Pixel coordinate of reference point
+  PC1_1   =   1.290551569736E-05 / Coordinate transformation matrix element
+  PC1_2   =  5.9525007864732E-06 / Coordinate transformation matrix element
+  PC2_1   =  5.0226382102765E-06 / Coordinate transformation matrix element
+  PC2_2   = -1.2644844123757E-05 / Coordinate transformation matrix element
+  CDELT1  =                  1.0 / [deg] Coordinate increment at reference point
+  CDELT2  =                  1.0 / [deg] Coordinate increment at reference point
+  CUNIT1  = 'deg'                / Units of coordinate increment and value
+  CUNIT2  = 'deg'                / Units of coordinate increment and value
+  CTYPE1  = 'RA---TAN'           / TAN (gnomonic) projection + SIP distortions
+  CTYPE2  = 'DEC--TAN'           / TAN (gnomonic) projection + SIP distortions
+  CRVAL1  =        5.63056810618 / [deg] Coordinate value at reference point
+  CRVAL2  =      -72.05457184279 / [deg] Coordinate value at reference point
+  LONPOLE =                180.0 / [deg] Native longitude of celestial pole
+  LATPOLE =      -72.05457184279 / [deg] Native latitude of celestial pole
+  RADESYS = 'ICRS'                / Equatorial coordinate system
+
+
+The following imports are generally useful:
+
+  >>> import numpy as np
+  >>> from astropy.modeling import models
+  >>> from astropy import coordinates as coord
+  >>> from astropy import units as u
+  >>> from gwcs import wcs
+  >>> from gwcs import coordinate_frames as cf
+
+The ``forward_transform`` is constructed as a combined model using `astropy.modeling`.
+The ``frames`` are subclasses of `~gwcs.coordinate_frames.CoordinateFrame`. Although strings are
+acceptable as ``coordinate_frames`` it is recommended this is used only in testing/debugging.
+
+Using the `~astropy.modeling` package create a combined model to transform
+detector coordinates to ICRS following the FITS WCS standard convention.
+
```

Comment:
It would be helpful to have a general diagram of the transformation like:

```
image coordinate -> shifted -> rotated -> unprojected-> rotated on the sky
```

This could help understanding  the order in the next paragraph ("First, create a transform which shifts the input...").

EDIT: after reviewing `index.rst` I see that what I suggest here was largely done there. I just do not understand why they are not together (example + description).

Reply:
Are you saying it should be duplicated (almost) in fits_analog.rst?

Reply:
No. I was saying, that initially (at the time of the initial comment), I did not see that an overview was done in `index.rst`. Therefore my "edit" can be interpreted as "ignore my original comment" (unless you want to restructure things).